test/default_mounts.bats

```diff
@@ -7,6 +7,9 @@ function teardown() {
 }
 
 @test "bind secrets mounts to container" {
+    if test -n "$UID_MAPPINGS"; then
```

Comment:
it could.  I've added `TEST_USERNS` to pass the information down from the Makefile, `$UID_MAPPINGS` was already use by the tests

Reply:
Hmmm, would it make sense to be conditional on ``$UID_MAPPINGS`` and/or ``$GID_MAPPINGS`` then (i.e. use two vars instead of three)?  This may be helpful to someone in the future, if there's a need to test with different sets of mappings, or perhaps only UID or GID mappings.  I don't have a good sense of how useful that would/could be, so please take this suggestion with some salt.

Reply:
Is this just an issue of the secret mounts having the wrong UID on disk?